src/components/layout/HeaderMainTitle/HeaderMainTitle.test.js

```diff
@@ -8,7 +8,9 @@ describe("<HeaderMainTitle />", () => {
   test("it should mount and contain main title", () => {
     renderWithRoute(<HeaderMainTitle />);
 
-    expect(screen.getByText("NDC")).toBeInTheDocument();
-    expect(screen.getByText("National Data Catalog")).toBeInTheDocument();
+    expect(screen.getByRole("heading", { name: /ndc/i })).toBeInTheDocument();
```

Comment:
I think querying by logical roles is so much better! 👍 
That queue is a very cool resource, we should keep that in mind!